src/hooks/useGetSearchBooks.ts

```diff
@@ -0,0 +1,16 @@
+import { Items } from '../types/GoogleBooks'
+import { useGoogleBooksApi } from './useGoogleBooksApi'
+export const useGetSearchBooks = (initialParameter: string) => {
+  const { response, isLoading, isError, doFetch } = useGoogleBooksApi<Items>(
+    `/volumes?q=${initialParameter}`
+  )
+
+  return {
+    books: response?.items,
+    loading: isLoading,
+    error: isError,
+    searchBooks: (bookName: string) => {
+      doFetch(`/volumes?q=${bookName}`)
```

Comment:
DRY (DO NOT REPEAT YOURSELF). You can move that /volumens?q= string to a shared constant outside the hook

const VOLUMES_SEAECH_PATH = "/volumes?q="